test/plugin/scenarios/jetty-scenario/config/expectedData.yaml

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 segmentItems:
 - serviceName: jettyserver-scenario
-  segmentSize: 2
+  segmentSize: 3
```

Comment:
Why is this a specific value? I think it should be `gt 3` or something.
There are re-try mechanisms there, this determined value could cause unnecessary tests failing.

Reply:
has update, and I‘ve test ok local